x-pack/plugins/osquery/public/packs/pack_queries_status_table.tsx

```diff
@@ -157,7 +157,7 @@ function getLensAttributes(
         {
           $state: { store: FilterStateStore.APP_STATE },
           meta: {
-            indexRefName: 'filter-index-pattern-0',
+            index: 'filter-index-pattern-0',
             negate: false,
             alias: null,
             disabled: false,
```

Comment:
Not sure how to test these changes to the packs plugin.

Reply:
![Screenshot 2022-01-13 at 16 26 48](https://user-images.githubusercontent.com/16632552/149358794-2d7bc101-d9df-4cc3-a3c8-cf7141618874.png)
Seem to be working :+1 :)